Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -181,7 +181,11 @@ class BoolSetting: Setting {
         let control = UISwitch()
         control.onTintColor = UIConstants.ControlTintColor
         control.addTarget(self, action: #selector(BoolSetting.switchValueChanged(_:)), for: UIControlEvents.valueChanged)
-        control.isOn = prefs.boolForKey(prefKey) ?? defaultValue
+        if let key = prefKey {
+            control.isOn = prefs.boolForKey(key) ?? defaultValue
+        } else {
+            control.isOn = defaultValue
+        }
```

Comment:
Nit: this starts to get messy quickly. Avoid checking for `prefKey` by using the type system. e.g.

```swift
func displayBool(_ control: UISwitch) {
   control.isOn = prefs.boolForKey(key) ?? defaultValue
}

func writeBool(_ control: UISwitch) {
    prefs.setBool(control.isOn, forKey: prefKey)
}
```
 And in the `FeatureSwitch` variant:
```swift
func displayBool(_ control: UISwitch) {
   control.isOn = self.featureSwitch.isMember(prefs)
}

func writeBool(_ control: UISwitch) {
    self.featureSwitch.setMembership(control.isOn, for: prefs)
}
```